ecom-login/ecom-login.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ShopShip/Login</title>
+    <link rel="stylesheet" href="https://garuda-ui.netlify.app/index.css">
+    <link rel="stylesheet" href="/ecom-login/ecom-login.css">
+    <link rel="stylesheet" href="/home.css">
+    <script src="https://kit.fontawesome.com/108bfd668b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="e-com-header">
+        <nav class="e-com-navbar shop-nav ecom-bg-blue">
+            <div class="ecom-white">ShopShip</div>
+            <ul class="e-com-nav-items">
+            </ul>
+            <input type="text" class="search-input ecom-search" placeholder="search">
+            <div class="e-com-social">
+                <div class="ecom-login-container">
+                    <button class="login-btn ecom-bg-white ecom-blue">Login</button>
+                </div>
+                <a href="" class="e-com-nav-icons anchor-style pointer">
+                    <i class="fas fa-heart heart-icon ecom-white">
+                        <div class="ecom-badge">4</div>
+                    </i>
+                </a>
+                <a href="" class="e-com-nav-icons anchor-style pointer">
+                    <i class="fas fa-shopping-cart cart-icon ecom-white">
+                        <div class="ecom-badge">5</div>
+                        <span class="cart-text ecom-white">cart</span>
+                    </i>
+                </a>
+            </div>
+        </nav>
+    </header>
+    <section class="login-section centered align-items">
+        <div class="login-wrapper flex flex-direction-col align-items px-3 pb-4">
+            <h1 class="mb-3">Login</h1>
+            <div class="pb-1">
+            <label for="" class="email-label">Email address</label>
+            <div>
+                <input type="text" class="login-input px-3">
+            </div>
+            </div>
+            <div class="pb-1">
+            <label for="" class="passwaord-label">Password</label>
+            <div>
+                <input type="password" class="login-input px-3">
+            </div>
+            </div>
+            <div class="flex pb-1">
+                <input type="checkbox">
+                <div class="remember">Remember me</div>
+                <a class="ml-3" href="">Forgot your Password?</a>
```

Comment:
Try to align remember me and forgot password better. You can simply use forgot password? instead of Forgot your Password? as it will decrease the text.

Reply:
ok done